src/prefect/utilities/notifications/notifications.py

```diff
@@ -125,7 +125,12 @@ def slack_message_formatter(
     tracked_obj: TrackedObjectType,
     state: "prefect.engine.state.State",
     backend_info: bool = True,
+    template: str = None,
```

Comment:
`None` here so the default value can still be defined in this function but also updated from `slack_notifier` -- not great but preserves the existing location of the default message.